src/usePagenation/index.ts

```diff
@@ -0,0 +1,36 @@
+import { useCallback, useLayoutEffect, useRef, useState } from 'react';
+import CreatePagenation from './createPagenation';
+
+interface options {
+    dataKey?: String;
+    currentKey?: String;
+    sizeKey?: String;
+    totalKey?: String;
+    defaultCurrent?: String;
+    defaultPageSize?: String;
+    isAutostart?: Boolean;
+}
```

Comment:
By convention interfaces use Pascal case also each of these types are calling the constructors and not the types